docs/topics/mta-7-installing-web-console-on-openshift.adoc

```diff
@@ -157,6 +157,38 @@ The most commonly used CR settings are listed in this table:
 |`analyzer_container_requests_memory`
 |`4Gi`
 |Minimum amount of memory the pod needs to run
+
+|`ui_container_limits_cpu`
+|`500m`
```

Comment:
@dymurray && @istein1 - please can i ask for a check on this value given in the Jira ticket as it does not look correct?